app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -797,8 +797,8 @@ public static ReadingData parseData(int attempt, String tagId, byte[] data, Long
             //       getGlucose(new byte[]{data[(i * 6 + 125)], data[(i * 6 + 124)]});
```

Comment:
Please can you remove the comment if it it not necessary anymore?

Reply:
Done.